src/components/item/item.md.scss

```diff
@@ -135,6 +148,15 @@ $item-md-sliding-content-background: $list-md-background-color !default;
     background-position: right ($item-md-padding-right - 2) center;
     background-size: 14px 14px;
   }
+  
+  [dir="rtl"] .item-md[detail-push] .item-inner,
+  [dir="rtl"] button.item-md:not([detail-none]) .item-inner,
+  [dir="rtl"] a.item-md:not([detail-none]) .item-inner {
+    @include svg-background-image($item-md-detail-push-svg-rtl);
+    padding-right: 0;
+    padding-left: 32px;
```

Comment:
Well ... for now I've just added a variable at the top of the file like padding for divider, I've also name it right instead of end as everything else is right but I agree with you right should be end and left should be start, but this thing should change globally, am I right?